{
	"id": "CVE-2017-7479",
	"vulnerabilities": [
		{
			"id": "CVE-2017-7479",
			"description": "OpenVPN versions before 2.3.15 and before 2.4.2 are vulnerable to reachable assertion when packet-ID counter rolls over resulting into Denial of Service of server by authenticated attacker.",
			"severity": [
				{
					"type": "cvss_v30",
					"source": "security.alpinelinux.org",
					"cvss_v30": {
						"vector": "CVSS:3.0/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H",
						"base_score": 6.5,
						"base_severity": "MEDIUM",
						"temporal_score": 6.5,
						"temporal_severity": "MEDIUM",
						"environmental_score": 6.5,
						"environmental_severity": "MEDIUM"
					}
				}
			],
			"references": [
				{
					"source": "security.alpinelinux.org",
					"url": "https://security.alpinelinux.org/vuln/CVE-2017-7479"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "http://www.debian.org/security/2017/dsa-3900"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://community.openvpn.net/openvpn/wiki/QuarkslabAndCryptographyEngineerAudits"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "http://www.securityfocus.com/bid/98443"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "http://www.securitytracker.com/id/1038473"
				}
			],
			"published": "2017-05-15T18:29:00Z",
			"modified": "2024-06-30T13:00:21.002794Z"
		}
	],
	"detection": [
		{
			"ecosystem": "alpine:3.5",
			"vulnerable": true,
			"package": {
				"name": "openvpn"
			},
			"affected": {
				"type": "apk",
				"range": [
					{
						"lt": "2.3.15-r0"
					}
				],
				"fixed": [
					"2.3.15-r0"
				]
			}
		},
		{
			"ecosystem": "alpine:3.6",
			"vulnerable": true,
			"package": {
				"name": "openvpn"
			},
			"affected": {
				"type": "apk",
				"range": [
					{
						"lt": "2.4.2-r0"
					}
				],
				"fixed": [
					"2.4.2-r0"
				]
			}
		}
	],
	"data_source": "alpine-osv"
}
